etc/profile-m-z/retroarch.profile

```diff
@@ -0,0 +1,46 @@
+# Persistent local customizations
+include retroarch.local
+# Persistent global definitions
+include globals.local
+
+blacklist /boot
+blacklist /opt
+blacklist /usr/libexec
+blacklist /var
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.config/retroarch
+whitelist /usr/share/retroarch
+whitelist /usr/share/libretro
+
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
```

Comment:
Can we use wr and wvc?

Reply:
Why?

Reply:
Why not?

Reply:
It is fine with

```
include whitelist-common.inc
include whitelist-runuser-common.inc
include whitelist-usr-share-common.inc
include whitelist-var-common.inc
```

Reply:
and wrc?

Reply:
wrc is `include whitelist-runuser-common.inc`.

Reply:
Review comment says

> wrc: whitelist-run-common.inc

and whitelist-runuser-common.inc is wruc

Reply:
Yes, I applied it to retroarch.profile.